src/sentry/conf/server.py

```diff
@@ -842,6 +847,12 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
         "schedule": timedelta(hours=1),
         "options": {"expires": 3600},
     },
+    "dynamic-sampling-prioritize-projects": {
+        "task": "sentry.dynamic_sampling.tasks.prioritise_projects",
+        # Run daily at 08:00
+        "schedule": crontab(hour=8, minute=0),
```

Comment:
Should we not run this more frequently? One day is a lot of latency.

Reply:
i selected it randomly do you have any reasonable suggestions? E.g. every 8h or 4h?

Reply:
@jan-auer I'll fix it to every 8 hours for now.